colorcoder/ColorPairTest.java

```diff
@@ -0,0 +1,30 @@
+package colorcoder;
+
+public class ColorPairTest 

```

Comment:
Class name is ColorPairTest ....i see GetColorFromPairNumber and GetPairNumberFromColor methods in it. How does these function related to ColorPairTest....